src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,29 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        if (employees == null) {
+            throw new IllegalArgumentException("Input list cannot be null");
+        }
```

Comment:
Good practice to check for null input and throw an exception. Ensure that the `Employee` class has appropriate `getAge()` and `getName()` methods implemented.